### DISCUSSION TITLE
tabler in Angular or VueJS

### DISCUSSION THREAD

**Discussion by glysns** [tabler/tabler#1123] [Ideas]
Hello,
Excelente template, is it possible integrate in Angular or Vue.JS ?

Thanks

**Comment by rjd22** [comment]
@glysns I'm using tabler with vuejs3 and it integrates nicely since it's using bootstrap 5. 

You can make your own Vue components next to the bootstrap ones if you want to use the bootstrap JavaScript.

**Reply by odwrotnie** [reply]
Could you provide a simple example?

**Reply by rjd22** [reply]
@odwrotnie sorry there are no simple examples. This is the reason we don't add them to tabler. Integration with vuejs and react is a lot of work and takes quite some code.

**Reply by odwrotnie** [reply]
What's the simplest / default workflow for loading data from database / rest services?

**Comment by thesuhu** [comment]
i'm using tabler with angular, my problem is the **modal** not working with *ngif
